web/src/components/organisms/Settings/MyIntegrations/hooks.ts

```diff
@@ -0,0 +1,71 @@
+import { useCallback, useMemo, useState } from "react";
+
+import {
+  Integration,
+  IntegrationType,
+} from "@reearth-cms/components/molecules/MyIntegrations/types";
+import { useCreateIntegrationMutation, useGetMeQuery } from "@reearth-cms/gql/graphql-client-api";
+
+export default () => {
+  const [integrationModalShown, setIntegrationModalShown] = useState(false);
+  const { data } = useGetMeQuery();
+
+  const [createNewIntegration] = useCreateIntegrationMutation({
+    refetchQueries: ["GetMe"],
+  });
+
+  const integrations = useMemo(() => {
+    return (data?.me?.integrations ?? [])
```

Comment:
Please update anywhere you were doing this to keep things consistent with the data?.some?.thing?.map() style

Reply:
Changed, and I will change accordingly when I face other instances.